client/src/components/Visualization/Visualization.js

```diff
@@ -0,0 +1,38 @@
+import './Visualization.css';
+
+import { Col, Container, Row } from 'react-grid-system';
+
+import BarChart from './charts/BarChart';
+import Interpolation from './charts/Interpolation';
+import React from 'react';
+import ZoomLine from './charts/ZoomLine';
+import tardigrade from '../../images/tardigrade.png';
+
+const CHARTS_TABLE = [['Zoomable Line'], ['Interpolation', 'Vertical Bar']];
```

Comment:
What is the semantic meaning of this array's internal arrangement?  Add a comment.